bin/dev/create_helm_charts_pr.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is used to open a PR against this repository:
+# https://github.com/suse/kubernetes-charts-suse-com
+# It will fetch the bundle specified by CAP_BUNDLE and will update the helm
+# charts in the above repo.
+
+set -e
+
+if [ -z $GITHUB_USER  ]; then
+  echo "GITHUB_USER environment variable not set"
+  exit 1
+fi
+
+if [ -z $GITHUB_PASSWORD ]; then
+  echo "GITHUB_PASSWORD environment variable not set"
+  exit 1
+fi
+
+if [[ -z "${CAP_BUNDLE}" ]]; then
+  echo "CAP_BUNDLE not set"
+  exit 1
+fi
+
+TMP=$(mktemp -d -p $PWD)
+#trap "rm -rf ${TMP}" EXIT
+
+pushd $TMP
+
+# Get the "hub" cli
+# https://hub.github.com/hub.1.html
+wget https://github.com/github/hub/releases/download/v2.3.0-pre10/hub-linux-amd64-2.3.0-pre10.tgz
+tar xvzf hub-linux-*.tgz
+cp hub-linux-*/bin/hub hub
+HUB=$(readlink -f hub)
+chmod +x $HUB
+
+# escape user and password
+USER=$(ruby -e "require 'cgi'; puts CGI.escape(ENV['GITHUB_USER'])")
+PASS=$(ruby -e "require 'cgi'; puts CGI.escape(ENV['GITHUB_PASSWORD'])")
+# Clone the kubernetes-charts-suse-com github repo
+$HUB clone https://${USER}:${PASS}@github.com/SUSE/kubernetes-charts-suse-com.git
+
+mkdir bundle
+pushd bundle
+wget "$CAP_BUNDLE"
+bundle=$(ls *.zip)
+unzip $bundle
+popd
+
+if [ -d bundle/helm/cf-opensuse ]; then
+  suffix="-opensuse"
+fi
+
+cd kubernetes-charts-suse-com
+# Remove old charts
+rm -rf stable/cf${suffix:-}
+rm -rf stable/uaa${suffix:-}
+# Place the new ones
+cp -r ../bundle/helm/cf${suffix:-} stable/
+cp -r ../bundle/helm/uaa${suffix:-} stable/
+
+$HUB config user.email "cf-ci-bot@suse.de"
+$HUB config user.name "${GITHUB_USER}"
+$HUB checkout -b $bundle
+$HUB -c core.fileMode=false add .
+$HUB commit -m "Submitting "$bundlev 
```

Comment:
I don't think `$bundlev` is bound? Is this a typo?

Reply:
typo it is, fixing.